### PR TITLE
Fix service app packaging

### PR DESCRIPTION
**Pull request by armanbilge**
Alternative to https://github.com/gemini-hlsw/lucuma-odb/pull/481 and https://github.com/gemini-hlsw/lucuma-odb/pull/475, does this do what you want?

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -98,6 +98,7 @@ lazy val service = project
   .enablePlugins(NoPublishPlugin, JavaAppPackaging)
   .settings(
     name := "lucuma-odb-service",
+    projectDependencyArtifacts := (Compile / dependencyClasspathAsJars).value,
```
Comment (armanbilge) [inline build.sbt:101]: If this works I'll look into upstreaming it.

File: build.sbt [hunk 2]
```diff
@@ -98,6 +98,7 @@ lazy val service = project
   .enablePlugins(NoPublishPlugin, JavaAppPackaging)
```
Comment (armanbilge) [inline build.sbt:98]: I can't comment on it but the line above is suspicious to me.
```scala
.dependsOn(sequence % "compile->test", smartgcal % "compile->test")
```
This says that that `Compile` scope of the `service` module depends on the `Test` scope of `sequence` and `smartgcal`. Is that really right? I.e. you want all your test artifacts on your application classpath?

Comment (swalker2m) [reply build.sbt:98]: I think you're right.  Some additional test dependencies would need to be spelled out but I can update this in a future PR.

**Review by swalker2m — APPROVED** [review]
Thanks, looks great! Let's give it a try.  :+1:

**Pull request merged**